src/main/resources/ready-api-rest-api-spec.json

```diff
@@ -327,6 +327,92 @@
       }
     }
   },
+  "/v1/readyapi/executions/{executionID}/logs": {
```

Comment:
yeah - that would be nice - i'm guessing its on the SH roadmap but we'll have to do it manually for now...